modules/account-lib/src/utils/crypto.ts

```diff
@@ -1,4 +1,6 @@
 import { HDNode, ECPair, networks } from '@bitgo/utxo-lib';
+import { Seed } from 'algosdk';
+import * as stellar from 'stellar-sdk';
```

Comment:
Remove these 2 if they are not needed anymore

Reply:
I missed those sorry